### DISCUSSION TITLE
How to configure Prometheus running on kubernetis to otel-collector -config file

### DISCUSSION THREAD

**Discussion by RSoodAMAT** [open-telemetry/opentelemetry-collector#3478] [Q&A]
Hi ,

We are running Prometheus on Kubernetis cluster at 30090. It is a Node Port.
When i tried configuring the same with my local otel -collector config -demo file . I get the below exception

exporters:
prometheus:
endpoint: "k8s-m1:30090"
namespace: traincenter
const_labels:
label1: value1
logging:

Exception :

otel-collector_1 | Error: cannot setup pipelines: cannot start builtExporters: listen tcp 10.200.7.39:30090: bind: cannot assign requested address
otel-collector_1 | 2021/06/21 13:43:57 application run finished with error: cannot setup pipelines: cannot start builtExporters: listen tcp 10.200.7.39:30090: bind: cannot assign requested address

What is the issue .
Promethus is running on 9090 port with Node port being 30090.

**Comment by dashpole** [comment]
If the port is already in use by a NodePort service, you won't be able to bind to it here as well.  But I don't think that is your problem.  Is the collector running on the host network?  Normally, you would specify `Endpoint: localhost:9090` for your collector, and then use a similar node port service with 9090 mapped to 30090, instead of trying to bind directly to the node's port.
